docker/README.md

```diff
@@ -1,120 +1,177 @@
-   1. [Table of Contents](#table-of-contents)
-   2. [Prerequisites](#prerequisites)
-   3. [Running the Docker Image](#running-the-docker-image)
-      1. [Docker Run](#docker-run)
-      2. [Docker Compose](#docker-compose)
-      3. [Kubernetes Deployment](#kubernetes-deployment)
-   4. [Flag Breakdown](#flag-breakdown)
+  1. [Docker Setup](#docker-setup)
+  2. [Overview of celestia-app txsim](#overview-of-celestia-app-txsim)
+  3. [Prerequisites](#prerequisites)
+  4. [Quick-Start](#quick-start)
+  5. [Docker Compose](#docker-compose)
+  6. [Kubernetes Deployments](#kubernetes-deployment)
+  7. [Flag Breakdown](#flag-breakdown)
+
+## 🐳 Docker setup
+
+This documentation provides a step-by-step guide on how to start up a celestia
+app using a docker image. Docker provides a seamless setup for celestia-app
+in an isolated environment on your machine. With Docker,
+you do not have to worry about the manual configuration of the required
+dependencies, which can be a pain.
+
+## Overview of celestia app txsim
+
+The celestia-app `txsim` binary is a tool that can be
+used to simulate transactions on the Celestia network.
+The `txsim` Docker image is designed to run the `txsim` binary with a
+variety of configurable options.
 
 ## Prerequisites
 
-Before you can use the `txsim` Docker image, you must have a prefunded account set up. The `txsim` binary requires a prefunded account to function correctly. The keyring for this account should be stored in a file that can be accessed by the Docker container.
+- [Docker Desktop for Mac or Windows](https://docs.docker.com/get-docker) or
+[Docker Engine for Linux](https://docs.docker.com/engine/install/)
+and a basic understanding of Docker.
+
+- A prefunded account set up with the keyring stored in a file,
+to be accessed by an instance of the docker image.
 
-## Running the Docker Image
+## Quick-Start
 
-### Docker Run
+1. In your local machine, navigate to the home directory
 
-You can run the `txsim` Docker image using the `docker run` command. Here's an example:
+   ```bash
+   cd $HOME
+   ```
 
-```bash
-docker run -it -v ${HOME}/.celestia-app:/home/celestia ghcr.io/celestiaorg/txsim -k 0 -r http://consensus-validator-robusta-rc6.celestia-robusta.com:26657,http://consensus-full-robusta-rc6.celestia-robusta.com:26657 -g consensus-validator-robusta-rc6.celestia-robusta.com:9090 -t 10s -b 10 -d 100 -e 10
-```
+2. Create a file in which the keyring would be stored.
+The file would be mounted as a volume into the docker container.
 
-In this command, the `-v` option is used to mount the `${HOME}/.celestia-app` directory from the host to the `/home/celestia` directory in the Docker container. This allows the `txsim` binary to access the keyring for the prefunded account.
+   ```bash
+   touch .celestia-app
+   ```
 
-### Docker Compose
+3. Using a suitable text editor of your choice, open the
+.celestia-app file and paste the keyring of the prefunded account.
```

Comment:
I think it would be helpful here to tell people how they can setup the keyring using the `celestia-appd keys` subcommand. For example `celestia-appd keys add txsim` would create the key and print the mnemonic. That way you can either paste the mnemonic as a flag or copy the keyring directory across